### ISSUE TITLE
Accessibility - Main Steps Screen - Incorrect Search Button Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#32]
The "Main Steps" screen has incorrect Search button text. The text currently is "greater thanSteps" and should be "search".

**Comment by adriennerlong** [comment]
I've flagged this as a high priority issue. We will ask the developers to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Thanks, for getting the developers to fix this item. It's important for those using screen readers.

**Comment by jfc3-dol** [comment]
Thanks, for making the "Home" link say it's a home link.
